### ISSUE TITLE
CLI service tutorial needs to include a description of p3-submit-genome-annotation

### ISSUE THREAD

**Issue by ARWattam** [PATRIC3/patric3_website#2252]
A user (Philippe Chlenski) reports that "The PATRIC CLI services tutorial at 
https://docs.patricbrc.org/cli_tutorial/cli_services.html does not include 
a description of p3-submit-genome-annotation, although it is in the CLI."

**Comment by chris-c-thomas** [comment]
@olsonanl is this you?

**Comment by olsonanl** [comment]
I was hoping one could extrapolate from the assembly one to the annotation one but we should probably write one up. I think Philippe should draft a writeup :-)

**Comment by chris-c-thomas** [comment]
That works for me. @ARWattam feel free to let em know :)

**Comment by chris-c-thomas** [comment]
If the ```p3-submit-genome-assembly``` command behaves in the same way as the ```p3-submit-genome-annotation``` then I can write up a similar section within the guide. 

@olsonanl I noticed ```p3-submit``` is not included in the CLI reference page located here: https://docs.patricbrc.org/cli_tutorial/command_list/index.html

Is that intentional or should it be added. I'm not super familiar with using the CLI but I'm assuming you can submit most services through it?

**Comment by olsonanl** [comment]
Yes, it works pretty much exactly the same way. If you run it with -h you'll get the usage message.

It should be in the list of CLI scripts; I think Bruce was going to take a look at that.

**Comment by chris-c-thomas** [comment]
I'll defer to Bruce for this then.
